### ISSUE TITLE
Store - style trash pickup

### ISSUE THREAD

**Issue by jameskoster** [Automattic/wp-calypso#20854]
Just a few odds and ends that need tidying up;

- Trash action icon styling for shipping zones doesn't match products / promos etc
- Order status accessibility - I'm not convinced the colors meet WCAG AA guidelines for contrast
- Print and fulfil button alignment https://cloudup.com/cqB9YXxtKdJ
- Edit order - ragged right alignment https://cloudup.com/c2iRPRzd_hO
- "Applies" dropdown margin in promos https://cloudup.com/c_lWsqqKSaY
- Orders save button label = save. Promos + products = update. Should be consistent.
- Margin beneath "Payment authorisation" radios in PayPal settings (same applies for all payment methods) https://cloudup.com/cEVXKFyupx7
- Cash on delivery dialog is unnecessarily narrow https://cloudup.com/cCOAxTq6B91 let's make all methods the same
- Edit address link color in shipping & tax settings https://cloudup.com/clIv4LElp-q
- Selecting a location in Zones changes the dialog width https://cloudup.com/cWpq75Ph-WP

**Comment by kellychoffman** [comment]
Found one more: 
<img width="1280" alt="screen shot 2017-12-19 at 10 50 50 am" src="https://user-images.githubusercontent.com/5835847/34179099-d2b619be-e4c6-11e7-94e3-7a49103b48e3.png">

**Comment by ryelle** [comment]
I think all of these were addressed in https://github.com/Automattic/wp-calypso/pull/20856 – if anything is still outstanding, we can create a separate issue.
